WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -229,5 +232,36 @@ private static AppBuilder BuildAvaloniaApp()
 				.With(new AvaloniaNativePlatformOptions { UseDeferredRendering = true, UseGpu = true })
 				.With(new MacOSPlatformOptions { ShowInDock = true });
 		}
+
+		/// <summary>
+		/// Sets up and initializes the crash reporting UI.
+		/// </summary>
+		/// <param name="serializableException"></param>
+		/// <param name="logPath"></param>
+		/// <returns></returns>
+		private static AppBuilder BuildCrashReporterApp(SerializableException serializableException, string logPath)
```

Comment:
Does it make sense to merge the other builder with `IsCrashReporter` parameter and have only one method? Do it only if you feel it cleaner.

Reply:
Hmm.. no i think the 2 initialization paths are different enough since the crash reporter needs to 1) Have a fixed theme since we can't load the settings on global to detect the theme and 2) needs to set every renderer to sofware rendering mode to avoid complications in case the crash is of GPU nature.